test/common/fit_common.py

```diff
@@ -931,3 +1109,21 @@ def run_from_module(file_name):
     # Use this method in 'name == "__main__"' style test invocations
     # within individual test files
     run_nose(file_name)
+
+# determine who imported us.
+importer=inspect.getframeinfo(inspect.getouterframes(inspect.currentframe())[1][0])[0]
+if 'run_tests.py' in importer:
+    # we are being imported through run_tests.py (the fit wrapper)
+    # process sys.args as received by run_tests.py
+    compose_config(True)
+
+    # Bridge between old and new.  Remove when conversion complete
```

Comment:
Small comment for follow-on PRs.....Can the compose_global_config() be moved out of the if statement since it is done in both cases?